Global/Eclipse.gitignore

```diff
@@ -37,3 +37,25 @@ local.properties
 
 # TeXlipse plugin
 .texlipse
+
+# eclipse specific git ignore
```

Comment:
Please have a look at the top section of this file - there's a lot of duplicates in here...